zips/zip-0230.rst

```diff
@@ -222,86 +203,111 @@ Transaction Format
-|                                    |                          |                                                  |Action Group become invalid and should be rejected by consensus.     |
-+------------------------------------+--------------------------+--------------------------------------------------+---------------------------------------------------------------------+
-|``64 * nActionsOrchard``            |``vSpendAuthSigsOrchard`` |``byte[64 * nActionsOrchard]``                    |Authorizing signatures for each Action of the Action Group in a      |  
-|                                    |                          |                                                  |transaction.                                                         |
-+------------------------------------+--------------------------+--------------------------------------------------+---------------------------------------------------------------------+
-
-The encoding of ``OrchardZSAAction`` is described below.
++-----------------------------+------------------------------+------------------------------------------------+---------------------------------------------------------------------+
+| Bytes                       | Name                         | Data Type                                      | Description                                                         |
++=============================+==============================+================================================+=====================================================================+
+| varies                      |``nActionsOrchard``           |``compactSize``                                 |The number of Action descriptions in ``vActionsOrchard``.            |
+|                             |                              |                                                |This MUST have a value strictly greater than ``0``.                  |
++-----------------------------+------------------------------+------------------------------------------------+---------------------------------------------------------------------+
+| 852 × ``nActionsOrchard``   |``vActionsOrchard``           |``OrchardZSAAction[nActionsOrchard]``           |A sequence of OrchardZSA Action descriptions in the Action Group.    |
++-----------------------------+------------------------------+------------------------------------------------+---------------------------------------------------------------------+
+| 1                           |``flagsOrchard``              |``byte``                                        |As defined in Section 7.1 of the Protocol Specification              |
+|                             |                              |                                                |[#protocol-txnencoding]_.                                            |
```

Comment:
These should reference ZIP 225, or a more precise reference within the protocol spec, instead of section 7.1.